.github/workflows/phpunit.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       fail-fast: false
       matrix:
         php: [7.4, 8.0]
-        laravel: [6.*, 7.*, 8.*]
+        laravel: [7.*, 8.*]
```

Comment:
@eschricker however the test for L6 is still being removed here? 🤔

Reply:
Totally right. I have readded the tests.